### PR TITLE
Unicast model integration

### PR DESCRIPTION
**Pull request by gurjashanpannu**

### CODE REVIEW

File: src/veins/nodes/UnicastScenario.ned [hunk 1]
```diff
@@ -0,0 +1,61 @@
+//
```
Comment (mightyCelu) [inline src/veins/nodes/UnicastScenario.ned]: This does not seem to be a general purpose scenario that is likely to be reused. Could we move this file into the _example/unicast_ directory?

File: src/veins/modules/messages/WaveShortMessage.msg [hunk 2]
```diff
@@ -42,4 +42,8 @@ packet WaveShortMessage {
 	int recipientAddress = -1;
 	int serial = 0;
 	simtime_t timestamp = 0;
+
```
Comment (mightyCelu) [inline src/veins/modules/messages/WaveShortMessage.msg]: Is a new field to indicate a unicast message really necessary? With the existing recipientAddress, one could use a special value such as -1 to denote broadcasts with all other values being unicast addresses.

File: src/veins/modules/application/unicast/UnicastApplLayer.ned [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+// You should have received a copy of the GNU General Public License
+// along with this program; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+//
+
+package org.car2x.veins.modules.application.unicast;
+import org.car2x.veins.base.modules.IBaseApplLayer;
+
+//
+// WAVE application layer base class.
```
Comment (mightyCelu) [inline src/veins/modules/application/unicast/UnicastApplLayer.ned]: This comment does not seem to fit the contents

File: examples/unicast/unicast_static.ini [hunk 4]
```diff
@@ -0,0 +1,168 @@
+[General]
```
Comment (mightyCelu) [inline examples/unicast/unicast_static.ini]: The other example's ini-files are all named _omnetpp.ini_, could we rename this for consistency?

**Comment by sommer** [comment]
Already merged as PR #57

**Pull request closed without merging**